show/bgp_common.py

```diff
@@ -353,25 +354,29 @@ def show_routes(args, namespace, display, verbose, ipver):
     else:
         back_end_intf_set = None
     # get all the other arguments except json that needs to be the last argument of the cmd if present
+    # Handling of multi-asic show ip/v6 route summary will print directly FRR result of each namespace
+    check_summary_set = set(["sum", "summ", "summa", "summar", "summary"])
```

Comment:
Instead of adding the check for argument  is "summary" here.  Can we check if the args is empty  to aggregate the routes from all namesapces  for all other argument we can display the output as-is for each namespace ?

Reply:
We also like to aggregate the routes from all namespaces for the case where the user specified specific route or in json format.  So for these cases the argument will not be empty but we still like to combine the output from all namespaces and display it.  Are you saying that other than these cases we should always default to let each specified namespace FRR to handle those additional parameter options similar to the "summary" case?  There will be cases where the "json" option will also be honored and for those we will NOT combine the jason output but have each one displayed based on its own name space which will result to multiple json outputs depends on namespaces involved... Is this acceptable?  Or should we handle this "summary" as designed for now and make additional enhancement as we see there is a need to do so?

Reply:
@arlakshm I have made the changes to do as you suggested.  So other than the specific route specified and the json option for "show ip/v6 route" command such as "summary, table, tables,...etc" will now be all handled by the corresponding namespace FRR directly.  This should also be easier when SONiC decided to upgrade to higher FRR version in the future.